ext/app/electron/config.ts

```diff
@@ -0,0 +1,141 @@
+import * as electron from "electron";
+import * as fse from "fs-extra";
+import * as log from "app/server/lib/log";
+import * as net from 'net';
+import * as packageJson from "desktop.package.json";
+import * as path from "path";
+import bluebird from 'bluebird';
+import { commonUrls } from "app/common/gristUrls";
+
+const NO_VALIDATION = () => true;
+
+
+/**
+ * Copied from grist-core, since it is unsafe to import core code at this point.
```

Comment:
Thanks for bringing this again to my attention! A closer look suggests that the `TYPEORM_DATABASE` issue is an urban myth. I remember testing importing dbUtils before setting up `TYPEORM_DATABASE` and it didn't work, but it likely failed for some other reasons as I couldn't reproduce the failure now (and on the base commit as well). I just did more tests and as a conclusion it could be safely dismissed.

Will remove related hacks and comments. My bad for not catching it early on.

Reply:
Also, I read the dbUtils code, and logically believe that importing it is not supposed to have side effects.